README.md

```diff
@@ -109,45 +127,118 @@ Amoro contains modules as below:
+| Standard JDK 11 Build  | `mvn clean package`                       | Build all modules (except `amoro-mixed-trino`) with JDK 11 |  
+| JDK 11 with Hadoop 2.x | `mvn clean package -DskipTests -Phadoop2` | Build with JDK 11 and Hadoop 2.x dependencies              |  
+| JDK 11 Quick Build     | `mvn clean package -DskipTests`           | Skip tests and build with JDK 11                           |  
+
+#### Building with JDK 17
+
+For JDK 17, you can build all modules including `amoro-mixed-trino`:
+
+| Build Scenario         | Command                                                                                                                             | Description                                   |  
+|------------------------|-------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------|  
+| JDK 17 All Modules     | `mvn clean package -DskipTests -Ptoolchain,build-mixed-format-trino`                                                                | Build all modules including Trino with JDK 17 |  
+| JDK 17 Trino Only      | `mvn clean package -DskipTests -Pformat-mixed-format-trino,build-mixed-format-trino -pl 'amoro-format-mixed/amoro-mixed-trino' -am` | Build only Trino module with JDK 17           |  
+| JDK 17 with Hadoop 2.x | `mvn clean package -DskipTests -Ptoolchain,build-mixed-format-trino,hadoop2`                                                        | Build all modules with JDK 17 and Hadoop 2.x  |  
+
+### Basic Build Options
+
+The most commonly used build commands:
+
+| Build Scenario      | Command                                    | Description                                    |  
+|---------------------|--------------------------------------------|------------------------------------------------|  
+| Standard Build      | `mvn clean package`                        | Build all modules (except `amoro-mixed-trino`) |  
+| Quick Build         | `mvn clean package -DskipTests`            | Skip tests for faster build                    |  
+| Skip Frontend Build | `mvn clean package -Pskip-dashboard-build` | Skip Web Dashboard build                       |  
+| Hadoop 2.x Build    | `mvn clean package -DskipTests -Phadoop2`  | Use Hadoop 2.x dependencies (default is 3.x)   |  
+
+### Advanced Build Options
+
+<details>  
+<summary>Click to expand advanced build options</summary>  
+
+#### Special Configuration Builds
+
+| Build Scenario       | Command                                                    | Description                                       |  
+|----------------------|------------------------------------------------------------|---------------------------------------------------|  
+| Disable Disk Storage | `mvn clean package -DskipTests -Pno-extented-disk-storage` | Do not introduce RocksDB to avoid memory overflow |  
```

Comment:
Disk storage does not refer to the storage system. In addition, this configuration is rarely modified and can be omitted.